usb/usb_tool.js

```diff
@@ -18,10 +19,11 @@ const ws = new WebSocket(`ws://localhost:${port}`);
 
 let device = null;
 let parser = null;
+let is_arduino = false;
```

Comment:
This doesn't appear to be used anywhere. is it no longer used after my message regarding not worrying about Arduinos? If not used, it can be deleted.

If it's going to be used, can we either check to see if the current device _is_ an Arduino just prior to doing some Arduino-specific code or find another solution? Just trying to avoid too many globals.